package.json

```
@@ -43,7 +44,8 @@
     "lint-staged": ">=10",
     "prettier": "2.3.2",
     "stylelint": "^13.13.1",
-    "stylelint-config-standard": "^22.0.0"
+    "stylelint-config-standard": "^22.0.0",
+    "vite": "^2.4.2"
```

🔧 `vite` is already in the `dependencies` category

```suggestion
    "stylelint-config-standard": "^22.0.0"
```